edge-apps/welcome-app/src/stores/settings.ts

```diff
@@ -0,0 +1,78 @@
+import { ref } from 'vue'
+
+export interface WelcomeSettings {
+  welcome_heading?: string
+  welcome_message?: string
+  theme?: 'light' | 'dark'
+  override_locale?: string
+  override_timezone?: string
+  sentry_dsn?: string
+  screenly_color_accent?: string
+  screenly_color_light?: string
+  screenly_color_dark?: string
+  screenly_logo_light?: string
+  screenly_logo_dark?: string
```

Comment:
I believe these settings are already being handled in the base settings store. You can already remove them and use those from the base settings if possible.